tests/Tests/Skottie/AnimationTest.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using Xunit;
+using System.IO;
+
+namespace SkiaSharp.Tests
+{
+	public class AnimationTest : SKTest
```

Comment:
Could we add a test to call the various properties to make sure that all the marshalling is working correctly?